src/blog/2019-03-15/announcing-dotnet-framework-support.pug

```diff
@@ -1,100 +1,135 @@
+           For .NET Core projects this step can be skipped.      
+           For .NET Framework projects, make sure [nuget.exe](https://docs.microsoft.com/en-us/nuget/install-nuget-client-tools#windows) is installed on your system. 
+           Please follow their installation instructions.
 
-        ## Start using Stryker.NET
-        If you have been using Stryker.NET before, you might notice that the installation has changed. Please follow these steps to correctly upgrade and keep your Stryker up to date.
+        3. Remove old references
         
-        ### 1. Installation
-        #### Install stryker **globally***:
+           Remove the old `DotNetCliToolReference` from your test project since they won't be updated anymore.
 
-        ```
-        dotnet tool install -g dotnet-stryker
-        ```
+        4. Have fun using stryker!
+        
+            You can now kick off stryker using 
+            ```
+            dotnet stryker
+            ```
 
-    p *Global install needs .NET Core runtime 2.2+ to be installed
-    :markdown-it
+            On .NET Framework projects the solution file is required. You can provide this using:
+            ```
+            dotnet stryker --solution-path "../MySolution.sln"
+            ```
 
-        #### Install stryker **locally***: 
-        Create a file called dotnet-tools.json in your project folder. You should check this file into source control to make sure all team members have access to Stryker and other tools specified here.
+            If stryker is installed as a dotnet core 3.0 local tool, use the following instead:
 
-        ```
-        dotnet new tool-manifest
-        ```
+            ```
+            dotnet tool run dotnet-stryker
+            ```
 
-        Then install stryker without the -g flag while executing the following command in the project folder
+        ## ⏩ VSTest integration
 
-        ```
-        dotnet tool install dotnet-stryker
-        ```
+        Stryker.NET now hooks right into VSTest to execute your unit tests instead of running your tests from the command line.
+        This means a big performance boost, because it keeps the VSTest connection alive between test runs. 
+        Because Stryker.NET supports VSTest, this performance boost wil work for most test frameworks, including [XUnit](https://xunit.github.io/),
+        [NUnit](https://nunit.org/) and [MSTest](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-with-mstest).
 
-    p *Local install needs .NET Core runtime 3.0+ to be installed
-    :markdown-it
-        ### 2. Install dependencies
-        For .NET Core projects this step can be skipped.
-        
-        For .NET Framework projects, make sure [nuget.exe](https://docs.microsoft.com/en-us/nuget/install-nuget-client-tools#windows) is installed on your system. Please follow their installation instructions.
+        Special thanks to [@_dupdob_](https://twitter.com/_dupdob_) for all his hard work on this feature!
```

Comment:
While dupdob is a very active contributor and we appreciate his help very much, he did not actually contribute to the vstest integration so it's a bit weird to thank him for that :)

What he is working on is coverage analysis, which is mentioned further down in this article.

Reply:
Ok, removed his name 👍